Makefile

```diff
@@ -16,4 +16,3 @@ build:
 
 test:
 	@go test -v -race ./...
-
```

Comment:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
hmm interesting because it is result of end-of-file-fixer hook from https://github.com/pre-commit/pre-commit-hooks :)

this hook have description:

```
id: end-of-file-fixer
    name: fix end of files
    description: ensures that a file is either empty, or ends with one newline.
```
other files *.go in commits have removed last line, so we have a proof that hook not works well :(

Reply:
ok @chavacava i removed commit related to end of files, and leave related with trailing white spaces :)